src/zrc/api/serializers.py

```diff
@@ -333,6 +334,31 @@ class Meta:
             }
         }
 
+    def _get_resultaat_type(self, resultaat_type_url):
+        if not hasattr(self, '_resultaat_type'):
+            self._resultaat_type = None
+            if resultaat_type_url:
+                Client = import_string(settings.ZDS_CLIENT_CLASS)
+                client = Client.from_url(resultaat_type_url)
+                client.auth = APICredential.get_auth(
+                    resultaat_type_url,
+                    scopes=['zds.scopes.zaaktypes.lezen']
+                )
+                self._resultaat_type = client.request(resultaat_type_url, 'resultaattype')
```

Comment:
Please always use the most-specific operations as possible. `client.request` is only intended to be used if there's no other option.

```suggestion
                self._resultaat_type = client.retrieve('resultaattype', url=resultaat_type_url)
```

Reply:
copypast thing. Fixed